Feature/CLI-67/signin component markup

Howardk97 opened the pull request:
Please review my code for the markup for the sign in page.

I added 3 dependencies of fontawesome (svg, react, and brand icons) one dependency of google fonts through next/font, and classnames to input multiple classnames for a single element.

ari-mediavine commented on app/pages/login/index.js:
@Howardk97 Lets not map the Icons so we can have specific linking on it later. If anything lets make an SSO component that contains the font awesome icons for organization c:

apatwary12 commented on app/pages/login/index.js:
@Howardk97  Lets not map the Icons so we can have specific linking on it later. If anything lets make an SSO component that contains the font awesome icons for organization c:

apatwary12 commented:
@Howardk97 I'm also getting a Mixed Packaged error if so could you regenerate your dependencies with Yarn instead of NPM? 
Also can you convert the css to tailwind and remove all of the css files except for global?

Howardk97 commented:
Thank you for the helpful suggestions. I created an SSO component and rearranged the code in the index for the login page for organization purposes. Please let me know what you think.

apatwary12 commented:
@Howardk97 Siiickk Looks great! All thats left to do in this is: 

- [ ]  Convert all CSS besides global into tailwind. 
- [ ]  Check mix package issue. Dependency loading from either npm or yarn but not both c: 
- [ ] remove console logs in login component

Howardk97 commented:
Converted the css files to tailwind and removed the console logs. Can you describe more about the mix package issue? I was using yarn to install all dependencies. Was I supposed to use npm?

apatwary12 commented:
@Howardk97 I'm not seeing any updated commits on the tailwind. and are you getting a mixed package on yarn dev at all?

Howardk97 commented:
Sorry thought it pushed and see now that it didn't push correctly. Just pushed it now and checked that it is there. I do not see a mixed package. Where is that showing up on your side?

apatwary12 commented:
@Howardk97 I'll check the mixed package stuff on my end and commit to this PR c: is there a way we could double check the fonts with Matt or coutney to know what all font sizing we will be using so we're not loading in fonts in the components?

apatwary12 commented:
@Howardk97 I'm seeing this error for not using the built in font dependency 

`- warn Your project has `@next/font` installed as a dependency, please use the built-in `next/font` instead. The `@next/font` package will be removed in Next.js 14. You can migrate by running `npx @next/codemod@latest built-in-next-font .`. Read more: https://nextjs.org/docs/messages/built-in-next-font`


Also for tailwind lets use as many of the default classes we can out of the box so we're not overloading the use of single use classes. I.E. 
my-[10px] could be h-2.5


If you go into https://tailwindcss.com/docs/ you can see all the defaults tailwind is auto loaded with. 

Lets configure Defaults with Tailwind too https://tailwindcss.com/docs/adding-custom-styles 
Lets add base styles for 
h1, h2, h3, h4, h5, h6, and p tags so we're not custom adding it to everything. 
https://csstailwind.com/how-to-add-custom-css-or-classes-in-tailwindcss/#:~:text=You%20can%20add%20your%20own%20styles%20in%20TailwindCSS%20in%202,relevant%20layer%20using%20TailwindCSS%20directives.

apatwary12 reviewed (approved):
Looks Great gonna make two Tech Debt Tasks for 
getting defaults and setting up global set fonts
